tests/testthat/test-install.R

```diff
@@ -233,10 +241,10 @@ test_that("toolchain checks without fixes on Windows with RTools 4.0 work", {
 
 test_that("clean and rebuild works", {
   skip_on_cran()
-  if (os_is_windows()) skip_on_covr()
   expect_output(
     rebuild_cmdstan(),
-    paste0("CmdStan v", latest_released_version(), " built")
+    paste0("CmdStan v", cmdstan_version(), " built"),
+    fixed = TRUE
```

Comment:
Adding missing fixed = TRUE seems to have fixed all tests that failed only on covr so I removed all remaining skip_on_covr().